src/utils.rs

```diff
@@ -114,6 +114,45 @@ impl Color {
     }
 }
 
+/// A Color used in a gradient
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
+pub struct ColorGradient {
```

Comment:
Not a fan of this name, as it implies that `Gradient` is the noun here, when it's more like a `GradientColor`? Why can't you use the existing `Color` type here? How does support for gradients work on consoles that don't support as many colors?

Reply:
> Not a fan of this name, as it implies that Gradient is the noun here, when it's more like a GradientColor?

Indeed, 3 AM me was not the wisest on the naming.

> Why can't you use the existing Color type here? 

Adding another entry to the `Color` enum would not be backwards compatible with `match` statement without a default branch.

>How does support for gradients work on consoles that don't support as many colors?

At the time of your comment, badly. As of [bac82c](https://github.com/console-rs/console/commit/bac82cba0f537297eeb5a8b6ea8ef29aab150f08), it will simply not use the gradient colors.